BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.m

```diff
@@ -104,6 +104,7 @@
 BOXAPIItemType *const BOXAPIItemTypeCollaboration = @"collaboration";
 BOXAPIItemType *const BOXAPIItemTypeGroup = @"group";
 BOXAPIItemType *const BOXAPIItemTypeFileVersion = @"file_version";
+BOXAPIItemType *const BOXAPIItemTypeRecentItem = @"recent_item";
```

Comment:
Not directly right now, but may be worth keeping around for some purpose in the future.